src/vcpkg-test/messages.cpp

```diff
@@ -1,12 +1,37 @@
 #include <catch2/catch.hpp>
 
+#include <vcpkg/base/json.h>
 #include <vcpkg/base/messages.h>
 
 #include <vcpkg/commands.generate-message-map.h>
 
 using namespace vcpkg;
 using namespace vcpkg::Commands;
 
+TEST_CASE ("get path to locale from LCID", "[messages]")
+{
+    // valid LCID; Chinese
+    auto res = msg::get_locale_path(2052); 
+    CHECK(res == "locales/messages.zh-Hans.json");
+
+    // invalid LCID; default to English
+    res = msg::get_locale_path(0000); 
+    CHECK(res == "locales/messages.en.json");
+}
+ TEST_CASE ("get message_map from LCID", "[messages]")
+{
+     StringView msg_name = "AddCommandFirstArg";
+
+     // valid lcid; Spanish
+     auto map = msg::get_message_map_from_lcid(1034);
+     auto msg = map.get()->get(msg_name);
+     CHECK(msg->string(VCPKG_LINE_INFO) == "El primer par\u00e1metro que se va a agregar debe ser \"artefacto\" o \"puerto\".");
```

Comment:
```suggestion
     CHECK(msg->string(VCPKG_LINE_INFO) == "El primer par\u00e1metro que se va a agregar debe ser \"artefacto\" o \"puerto\".");
```

Not an issue for this PR but `"artefacto`" y `"puerto"` should be untranslated. Also, there is a typo, it should be `"artefacto"`...

Reply:
I noticed this as well. I was planning on reaching out to the loc team since they are the ones that do the translations. However, if this is happening for the Spanish translation, I can imagine it happening in other languages as well.